osi_logicaldetectiondata.proto

```diff
@@ -0,0 +1,230 @@
+
+        // Data is available.
+        //
+        DATA_QUALIFIER_AVAILABLE = 2;
+
+        // Reduced data is available.
+        //
+        DATA_QUALIFIER_AVAILABLE_REDUCED = 3;
+
+        // Data is not available.
+        //
+        DATA_QUALIFIER_NOT_AVAILABLE = 4;
+
+        // Sensor is blind.
+        //
+        DATA_QUALIFIER_BLINDNESS = 5;
+
+        // Sensor temporary available.
+        //
+        DATA_QUALIFIER_TEMPORARY_AVAILABLE = 6;
+
+        // Sensor invalid.
+        //
+        DATA_QUALIFIER_INVALID = 7;
+    }
+}
+
+//
+// \brief A logical detection that could be based on multiple sensors and sensor types.
+//
+message LogicalDetection
+{
+    // Existence probability of the logical detection
+    //
+    // \note Use as confidence measure where a low value means less confidence
+    // and a high value indicates strong confidence.
+    //
+    // \rules
+    // is_greater_than_or_equal_to: 0
+    // is_less_than_or_equal_to: 1
+    // \endrules
+    //
+    optional double existence_probability = 1;
+
+    // ID of the detected object this logical detection is associated to.
+    //
+    // \note ID = MAX(uint64) indicates no reference to an object.
+    //
+    // \rules
+    // refers_to: DetectedObject
+    // \endrules
+    //
+    optional Identifier object_id = 2;
+
+    // Measured position of the logical detection given in cartesian coordinates in the
+    // host vehicle coordinate system.
+    //
+    // Unit: m
+    //
+    optional Vector3d position = 3;
+
+    // Root mean squared error of the measured position of the logical detection.
+    //
+    optional Vector3d position_rmse = 4;
+
+    // Velocity of the logical detection given in cartesian coordinates in the
+    // host vehicle coordinate system.
+    //
+    // Unit: m/s
+    //
+    optional Vector3d velocity = 5;
+
+    // Root mean squared error of the logical detection's velocity.
+    //
+    // Unit: m/s
+    //
+    // \rules
+    // is_greater_than_or_equal_to: 0
+    // \endrules
+    //
+    optional Vector3d velocity_rmse = 6;
+
+    // Intensity or equivalent value of the logical detection's echo.
+    //
+    // Unit: %
+    //
+    // \rules
+    // is_greater_than_or_equal_to: 0
+    // is_less_than_or_equal_to: 100
+    // \endrules
+    //
+    optional double intensity = 7;
+
+    // The signal to noise ratio (SNR) of the logical detection.
+    //
+    // Unit: dB
+    //
+    optional double snr = 8;
+
+    // Describes the possibility whether more than one object may have led to
+    // this logical detection.
+    //
+    // \rules
+    // is_greater_than_or_equal_to: 0
+    // is_less_than_or_equal_to: 1
+    // \endrules
+    //
+    optional double point_target_probability = 9;
+
+    // The IDs of the sensors that have been fused to the single logical detection.
+    //
+    // \note one logical detection can originate from multiple sensors.
+    //
+    repeated Identifier sensor_id = 10;
```

Comment:
@PhRosenberger should be updated here as well :)

Reply:
Done.

Reply:
Perfect, thanks!